frame/uniques/src/tests.rs

```diff
@@ -695,15 +695,46 @@ fn max_supply_should_work() {
 	});
 }
 
+#[test]
+fn try_increment_id_works() {
+	new_test_ext().execute_with(|| {
+		// should fail because the next `CollectionId` is not being used.
+		assert_noop!(Uniques::try_increment_id(Origin::signed(2)), Error::<Test>::NextIdNotUsed);
+
+		// create two collections.
+		assert_ok!(Uniques::force_create(Origin::root(), 1, true));
+		assert_ok!(Uniques::force_create(Origin::root(), 1, true));
```

Comment:
You could also check for the `NextCollectionIdIncremented` events after these calls.